bindings/wasm/bindings.ts

```diff
@@ -0,0 +1,43 @@
+type Matrix1x3 = [Number, Number, Number];
+type Matrix4x3 = [Matrix1x3, Matrix1x3, Matrix1x3, Matrix1x3];
+type Vec3 = [Number, Number, Number];
```

Comment:
I believe you want the lower-case version of `number`: https://stackoverflow.com/a/42654496

Reply:
Ah this is interesting, I did not check this because it did not complain about the examples. Will fix.